src/components/app/Home.css

```diff
@@ -240,23 +240,28 @@
   opacity: 0;
 }
 
+.photon-input.homeSectionLoadFromUrlInput {
+  height: 13px;
+
+  /* the chain class selectors are used here 
+  to override the styling for the photon-input class */
+  padding: 8px 20px;
+}
```

Comment:
Actually I think this looks pretty good (even better) when removing this. What do you think?